src/Illuminate/Foundation/Console/DownCommand.php

```diff
@@ -42,21 +43,19 @@ class DownCommand extends Command
     /**
      * Execute the console command.
      *
+     * @param  \Illuminate\Foundation\MaintenanceMode  $maintenanceMode
      * @return int
      */
-    public function handle()
+    public function handle(MaintenanceMode $maintenanceMode)
     {
         try {
-            if (is_file(storage_path('framework/down'))) {
+            if ($maintenanceMode->isDown()) {
```

Comment:
```suggestion
            if ($this->laravel->maintenanceMode()->isDown()) {
```

Reply:
@driesvints You were one step ahead of me :). I've replace the DI usage with the application method. That begins to beg the question, now the arguments have been removed from the Middleware and the commands, should this PR be retarget to the 8.x branch instead of the master branch?